src/reuse/modules/ui5/types/ui5.types.ts

```diff
@@ -0,0 +1,9 @@
+export type Ui5Selector = {
+  elementProperties: any
+};
+export type Ui5SelectorOptions = {
```

Comment:
@DIFSRIP should we rename SelectorOptions?

Reply:
agree. @heraldantony Can you change to "Ui5SelectorWithOptions" please?

Reply:
yes, changed to Ui5SelectorWithOptions